src/generate.ts

```diff
@@ -1,27 +1,35 @@
-import * as zg from "zapatos/generate"
-import {
-  getConnectionStringFromEnv,
-  getPgConnectionFromEnv,
-} from "pg-connection-from-env"
-import { Context } from "./get-project-context"
-import { dumpTree } from "pg-schema-dump"
-import path from "path"
+import * as zg from "zapatos/generate";
+import { Context } from "./get-project-context";
+import { dumpTree } from "pg-schema-dump";
+import path from "path";
+import { PGlite } from "@electric-sql/pglite";
+
+class MockClient {
+  constructor(private pglite: PGlite) {}
+
+  async query(sql: string, params?: any[]) {
+    return this.pglite.query(sql, params);
+  }
+
+  async connect() {} 
+
+  async end() {
+    await this.pglite.close();
+  }
+}
```

Comment:
Not sure if we will need the class, we can just query the pglite after creating ths pglite instance